libs/auth/scripts/src/utils.ts

```diff
@@ -38,18 +40,16 @@ export async function waitForReadyCardStatus<T>(
 /**
  * Programs a system administrator Java Card
  */
-export async function programSystemAdministratorJavaCard({
+export async function programJavaCard({
   card,
-  isProduction,
-  jurisdiction,
+  user,
 }: {
   card: JavaCard;
-  isProduction: boolean;
-  jurisdiction: string;
+  user: VendorUser | SystemAdministratorUser;
```

Comment:
Prepping this helper for use in the vendor card creation script, introduced in the next commit